plugin/storage/grpc/proto/storage.proto

```diff
@@ -0,0 +1,135 @@
+syntax = "proto3";
+
+package jaeger.storage.v1;
+
+option go_package = "api_v1";
+
+import "gogoproto/gogo.proto";
+import "google/protobuf/timestamp.proto";
+import "google/protobuf/duration.proto";
+import "google/protobuf/empty.proto";
+
+import "model.proto";
+
+// Enable gogoprotobuf extensions (https://github.com/gogo/protobuf/blob/master/extensions.md).
+// Enable custom Marshal method.
+option (gogoproto.marshaler_all) = true;
+// Enable custom Unmarshal method.
+option (gogoproto.unmarshaler_all) = true;
+// Enable custom Size method (Required by Marshal and Unmarshal).
+option (gogoproto.sizer_all) = true;
+// Enable registration with golang/protobuf for the grpc-gateway.
+option (gogoproto.goproto_registration) = true;
+
+message GetDependenciesRequest {
+    google.protobuf.Timestamp end_timestamp = 1 [
```

Comment:
minor, but I think we should clean up the API and use `start_time` and `end_time` (don't need to call them `timestamp`, it's just the type of the value, not the meaning of it).